vmdb/app/models/miq_provision_vmware/customization.rb

```diff
@@ -266,7 +266,6 @@ def set_spec_array_option(obj, property, key, override_value=nil)
     if key.nil?
       value = get_option(nil, override_value)
     else
-      value = override_value.nil? ? get_option(key) : override_value
       value = get_option(key)
```

Comment:
Yay!  I'll update this with @gmcculloug's advice.

For reference, there's many results of the UselessAssignment cop... I did only a tiny number.

`rubocop -f e --only Lint/UselessAssignment`